src/components/VmUserMessages/index.js

```diff
@@ -66,6 +66,26 @@ class VmUserMessages extends React.Component {
       show = 'show'
     }
 
+    let messages = userMessages.get('records')
+    let messagesList = ''
+
+    if (messages.size !== 0) {
+      messagesList = (
+        <ul className={`list-group ${style['messages-list']}`}>
+          {messages.map(r => (<UserMessage key={r.time} record={r} id={`${idPrefix}-msg-${idCounter++}`} />))}
+        </ul>)
+    } else {
+      messagesList = (
+        <div className='blank-slate-pf '>
```

Comment:
Could you please add custom style rules `border` and `magin-bottom` for this element to remove unnecessary white space and border line?

![image](https://user-images.githubusercontent.com/5278760/32981450-385d3a7c-cc76-11e7-8696-97e0806635bb.png)